### DISCUSSION TITLE
custom plot color

### DISCUSSION THREAD

**Discussion by lgarcia-ebucis** [highfestiva/finplot#177] [Q&A] [answered]
Hi, 

is there a way to provide a custom color function on scatter plot? something like colorfunc on candlestick_ochl? 

Thank you

**Comment by highfestiva** [comment, marked as answer]
Unfortunately not, it's the same as with line plots. You'll have to create a new scatter plot for each color you want displayed. If you have ten colors, you'll have to use ten plots. You could easily wrap them in a function of your own, and just loop over each color and plot.

**Reply by lgarcia-ebucis** [reply]
thanks
